app/src/main/java/com/example/hackernews/ArticleFragment.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.hackernews
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.WebViewClient
+import android.widget.Toast
+import androidx.activity.OnBackPressedCallback
+import androidx.navigation.NavController
+import androidx.navigation.fragment.findNavController
+import com.example.hackernews.databinding.FragmentArticleBinding
+import com.example.hackernews.interfaces.Navigable
+
+private const val ARG_PARAM1 = "URL"
+
+class ArticleFragment : Fragment(), Navigable {
+
+    private var param1: String? = null
```

Comment:
Property name should reflect its content e.g. `private var url: String?`. Also, there is no need to explicitly initialize it to null.